src/js/hca-rjsf/hca-rjsf-entry.jsx

```diff
@@ -0,0 +1,35 @@
+import 'core-js';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createHistory } from 'history';
+import { Router, useRouterHistory } from 'react-router';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+import initReact from '../common/init-react';
+import createRoutes from './routes';
+import reducer from './reducer';
+
+require('../common');
+require('../../sass/hca.scss');
+
+require('../login/login-entry.jsx');
+
+const store = createStore(reducer);
```

Comment:
May want to apply the middleware like the edu forms do:

```js
let store;
if (__BUILDTYPE__ === 'development' && window.devToolsExtension) {
  store = createStore(reducer, compose(applyMiddleware(thunk), window.devToolsExtension()));
} else {
  store = createStore(reducer, compose(applyMiddleware(thunk)));
}
```

I don't know if `redux-thunk` is necessary, strictly speaking, with the schemaform, but maybe @jbalboni can weigh in on that. At any rate, the react dev tools might be nice.